docker/package/fedora.py

```diff
@@ -20,7 +20,7 @@ def build_fedora_package(
     home = os.environ["HOME"]
 
     pkg.fetch_sources(dir)
-    pkg.gen_makefile(f"{dir}/Makefile")
+    pkg.gen_buildfile("/".join([dir, pkg.buildfile]))
```

Comment:
Do I understand correctly that `rpmbuild` is capable of using `setup.py` instead of `Makefile` out of the box?